packages/hint-manifest-icons/src/hint.ts

```diff
@@ -17,6 +17,9 @@ import { extname } from 'path';
 
 const debug: debug.IDebugger = d(__filename);
 
+// Valid values for icon purpose property.
+const validPurposes = ['any', 'maskable', 'badge', 'monochrome'];
```

Comment:
as w3c/manifest#833 has been merged, the spec does not mention `badge` any more, so i think it should be removed from `validPurposes`, though maybe there should be a different issue dealing with this? (especially because you might want webhint to have a specific behaviour, like warning `badge` was replaced?)

Reply:
@sarvaje @hxlnt this hint is still not [published "officially"](https://github.com/webhintio/hint/blob/145ab5a11f7077a63b28a954f9a45bc5efa20e9b/packages/hint-manifest-icons/package.json#L55) although it seems we are using it seems to be active in different places. IIRC the missing piece was to update the documentation. Ideally we should do the `badge/monochrome` before we "release" it.

https://github.com/w3c/manifest/pull/833#issuecomment-645756918

> Confirmed that Chrome does not use purpose: badge anywhere, including on the server-side infrastructure. From our perspective, we are good to rename it and change its semantics. LGTM.

@amazingcaio, thanks for catching this change!

Reply:
Thanks @amazingcaio!

Reply:
no problem. also, there were a couple of places where you misspelt `property` as `porperty`. you may want to fix that :)

Reply:
Thanks!! It should be fixed now.